src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx

```diff
@@ -769,13 +782,19 @@ describe("ConfirmBid for unqualified user", () => {
         </BiddingThemeProvider>
       )
 
-      // manually setting state to avoid duplicating tests for skipping UI interaction, but practically better not to do this.
-      component.root.instance.setState({ billingAddress })
-      component.root.instance.setState({ creditCardToken: stripeToken })
-      component.root.findByType(Checkbox).instance.props.onPress()
-      component.root.findByType(Button).instance.props.onPress()
+      fillOutFormAndSubmit(component)
 
-      jest.runAllTicks()
+      expect(relay.commitMutation).toHaveBeenCalled()
+      expect(relay.commitMutation).toHaveBeenCalledWith(
+        expect.any(Object),
+        expect.objectContaining({
+          variables: {
+            input: {
+              phone: "111 222 4444",
+            },
+          },
+        })
+      )
```

Comment:
The `updatePhoneNumber()` call also seems like a new mutation, so adding it to the happy path test.